File: C/String/program3.c
//Display the string

#include<stdio.h>

int main()
{
    char Arr[] = "Hello";
    int i = 0;

    printf("The string is: \n");
    
    for(i = 0; Arr[i] != '\0'; i++)
    {
        printf("%c",Arr[i]);
    }

    return 0;
}

File: Java/String/program3.java
//Accept the string and display each character

import java.util.*;

class program3
{
    public static void main(String arg[])
    {
        String str = "Hello";
        
        char Arr[] = str.toCharArray();

        int i = 0;

        while(i < Arr.length)
        {
            System.out.println(Arr[i]);
            i++;
        }
    }
}